### ISSUE TITLE
Compatibility with Precision Farming UI

### ISSUE THREAD

**Issue by Dwinesk** [jtsage/FS22_simpleInspector#32]
Alignment issue
The newest update allows one to have the FS22_simpleInspector hud underneath the Precision Farming UI. Issue is though that if applying lime or fertilizer, once on a field, the Precision Farming UI expands and your mod does'nt account for that

**To Reproduce**
1. Using Precision Farming
2. Have FS22_simpleInspector in the Top Left Corner
3. Go Lime or Fertilizing in a field

**Expected behavior**
FS22_simpleInspector Hud to move down to avoid obscuring Precision Farming UI

**Screenshots**
![20230705032955_1](https://github.com/jtsage/FS22_simpleInspector/assets/29665879/03b25391-e5cf-443b-bd5e-26e339a2c6dc)

![20230705032712_1](https://github.com/jtsage/FS22_simpleInspector/assets/29665879/dddffb8b-3b77-42ea-978f-dc1b8ea8b29a)

**Desktop :**
 - OS: Win 11
 - Browser: Chrome
 - Version  114.0.5735.199

**Comment by Dwinesk** [comment]
Just seen that this overlaps with issue #28 so yeah.. here's more evidence of that.

**Comment by jtsage** [comment]
Yeah.  I've not stumbled on a good fix yet - the issue is that the "extra" bits of that overlay are drawn last in the chain, and it doesn't seem like the function that does it really knows or cares how big it is - still looking for a good fix.

**Comment by KITT3000** [comment]
@jtsage and it would not be possible to check the distance from the "Application rate" text/element, then with a fixed offset it would move your entire window by as much as the position of this text/element changes?

**Comment by jtsage** [comment]
That's the issue I'm having - *finding* the application rate (outside of the PF mod lua scope)  I may have seen where that's done in the PF stuff, but near as I can figure, it's not set up to be able to be referenced from outside of the PF code.

**Comment by Dwinesk** [comment]
@jtsage how about you try moving your entire UI down a few millimeters(as an exception) so that it will never be in the way when Precision Farming is active. And for checks you can use whether it is seeding, liming, fertilizing or the combination of both seeding and fertilizing?
